templates/index.html

```diff
@@ -5,18 +5,34 @@
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   </head>
   <body>
+  <center>
 
     <h1>So it begins...</h1>
 
-    <div id="pile0"></div>
-    <div id="pile1"></div>
-    <div id="pile2"></div>
-    <div id="pile3"></div>
-    <div id="pile4"></div>
+        <h3>
+        Enter Minimum stones per pile <br><input name="desired_min" type="text" size="5"> <br><br>
+        Enter Maximum stones per pile <br><input name="desired_max" type="text" size="5" > <br><br>
+        Enter Number of piles <br><input name="desired_piles" type="text" size="5"> <br><br>
+        </h3>
```

Comment:
The ticket said we should pass those to the backend. Did you put it up for review before getting to that? Are you planning on creating a separate ticket to handle that instead?

Reply:
Ah, so you read the ticket the same way I did. 
Here's how the events transpired: 
- Dave reads the ticket and thinks the scope includes passing JSON to the backend. 
- Dave tries to get this working, has it close but is scratching his head at not being able to get it working as it should. 
- Monday begins, Dave talks to Jesse about said ticket. 
- Jesse informs Dave that this is not at all in the scope of ticket 'nim-6', that the line talking about passing to the backend was simply a comment on what these input fields and button will be eventually used for. 

As far as I'm aware now, that will be dealt with as part of https://hacksaurz.atlassian.net/browse/NIM-16 and https://hacksaurz.atlassian.net/browse/NIM-15

These are now assigned to @dodecaphonix I believe. 
Does that all sound correct @dodecaphonix  ? 

*TLDR:*
I think that's outside the scope of this ticket. Jesse is handling it